src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -914,23 +982,23 @@ registerThemingParticipant((theme, collector) => {
 	}
 	const buttonHoverBackgroundColor = theme.getColor(buttonHoverBackground);
 	if (buttonHoverBackgroundColor) {
-		collector.addRule(`.monaco-workbench .part.editor > .content .welcomePageContainer .btn_primary:hover { background: ${buttonHoverBackgroundColor};}`);
+		collector.addRule(`.monaco-workbench .part.editor > .content .welcomePageContainer .btn_primary:hover { background: ${buttonHoverBackgroundColor} !important;}`);
```

Comment:
usually we will try to avoid using !important, why are we adding that here in this PR? that doesn't seem to be related to the issues we are fixing.